internal/k8s/controllers/service_controller.go

```diff
@@ -67,6 +68,11 @@ func (r *ServiceReconciler) reconcileService(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, err
 	}
 
+	if filterByLoadbalancerClass(service, r.LoadBalancerClass) {
```

Comment:
iiuc loadBalancerClass is immutable, can we use this in this controller as a predicate?